Containerfile

```diff
@@ -8,5 +8,7 @@ RUN R -e "install.packages(c('assist', 'date', 'geosphere', 'glmnet',  'tictoc',
 
 RUN R -e "install.packages('https://cran.r-project.org/src/contrib/Archive/zipcode/zipcode_1.0.tar.gz', repos=NULL, type='source', Ncpus = parallel::detectCores())"
 
+RUN mamba install -y -c pytorch transformers 
```

Comment:
Might be better to use pip for this as they are bundled together in one install there and that might possibly give a better chance for the versions of each to work together:

`pip install 'transformers[torch]'`

Ref: https://huggingface.co/docs/transformers/installation